utils/ccloud_library.sh

```diff
@@ -34,16 +34,16 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
 function ccloud::prompt_continue_ccloud_demo() {
   echo
-  echo "---------------------------------------------------------------------------------------------"
-  echo "Any Confluent Cloud example uses real Confluent Cloud resources that may be billable."
-  echo "An example may create a new Confluent Cloud environment, Kafka cluster, topics, ACLs,"
-  echo "and service accounts, as well as resources that have hourly charges like connectors and"
-  echo "ksqlDB applications."
+  echo "--------------------------------------------------------------------------------------------"
+  echo "Any Confluent Cloud example uses real Confluent Cloud resources that may be billable,"
+  echo "including connectors and ksqlDB applications that may have hourly charges."
   echo
-  echo "To avoid unexpected charges, carefully evaluate the cost of resources before you start."
-  echo "After you are done running a Confluent Cloud example, destroy all Confluent Cloud resources"
-  echo "to avoid accruing hourly charges for services and verify that they have been deleted."
-  echo "---------------------------------------------------------------------------------------------"
+  echo "Use Confluent Cloud promo code C50INTEG to receive \$50 free usage, which should"
+  echo "sufficiently cover one day of running this example, beyond which you may be billed"
+  echo "for the services until you destroy the resources."
+  echo
+  echo "Use the provided script to destroy all resources. Always verify that they have been deleted."
```

Comment:
FYC.  I'm just trying to inform the user where the the destroy command will be presented to them later so they aren't looking for it here while reading this.

```suggestion
  echo "After startup, this demo will provide a command to destroy all the demo resources. After running the command, use the Confluent Cloud UI or the `ccloud` CLI to verify the demo resources have been deleted."
```

Reply:
Good idea.  I hadn't done that at first because I wasn't sure every example did that, but now I checked and see that they do!  So, I will rework the text to capture this suggestion (but will fix line breaks).